nbqa/save_source.py

```diff
@@ -27,21 +28,32 @@ def _replace_magics(source: List[str]) -> Iterator[str]:
     ----------
     source
         Source from notebook cell.
+    ignore_cells
+        Extra cells which nbqa should ignore.
 
     Yields
     ------
     str
         Line from cell, possibly commented out.
     """
-    bash_cell = source and any(source[0].startswith(i) for i in ("%%script", "%%bash"))
+    if ignore_cells is not None:
+        ignore = MAGIC + ignore_cells.split(",")
```

Comment:
After strip, its better to strip leading and trailing whitespaces

Some users might provide `ignore_cells = %%html, %%cpython` in the configuration.

Reply:
I hadn't thought of that, thanks! This makes it a bit more user-friendly, and I can't see any drawbacks...

Reply:
@girip11 done, thanks for the idea!

Let's see if this works:
@all-contributors please add @girip11 for ideas